cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMapping.java

```diff
@@ -326,13 +296,14 @@ private boolean setRemoteRegulatingTerminal(String tc, RegulatingControl control
         return false;
     }
 
-    private boolean setRemoteRegulatingTerminal(RegulatingControl control, Generator g) {
-        Terminal regTerminal = findRemoteRegulatingTerminal(control.cgmesTerminal, control.topologicalNode);
-        if (regTerminal == null) {
-            context.missing(String.format(MISSING_IIDM_TERMINAL, control.topologicalNode));
-            return false;
-        }
-        g.setRegulatingTerminal(regTerminal);
-        return true;
+    public void setAllRegulatingControls(Network network) {
+        regulatingControlMappingForGenerators.applyRegulatingControls(network);
+
+        cachedRegulatingControls.clear();
```

Comment:
Don't forget to check at the end if regulating controls remain in the cache to write warning messages in the logger stating that they are not handled. It is not really relevant to do it now since the `setAllRemoteRegulatingTerminals` already does it but when it is deleted, we should still write it.

Reply:
Done